### ISSUE TITLE
Don't resize the OpenGL preview window to output pixel resolution values 

### ISSUE THREAD

**Issue by kilacoda** [ManimCommunity/manim#1437]
## Description of bug / unexpected behavior

Opening a scene with the `-p` flag launches a window, which scales itself according to the resolution values:
https://github.com/ManimCommunity/manim/blob/b57f0f401905646669dfef6060bc805b46c28ea9/manim/renderer/opengl_renderer_window.py#L15-L18

This can create unnecessarily large windows when using higher resolutions (say, `-qp`, or `-qk` etc.) which are difficult to manage, especially on small screens like those of laptops (`-qk` would give you a 3840x2160 pixel window!). Not everyone has a fancy widescreen monitor y'know.

## Expected behavior

Reasonably sized preview window, not dependent on the output media's resolution. Maybe a fixed size window as used in 3b1b/manim?

https://github.com/3b1b/manim/blob/master/manimlib/window.py#L17-L24

## How to reproduce the issue

Use any scene, just render with something like `-pqp --renderer=opengl`
<details><summary>Code for reproducing the problem</summary>

```py
Paste your code here.
```

</details>

## Additional media files

<details><summary>Images/GIFs</summary>

</details>

## Logs
<details><summary>Terminal output</summary>

```
PASTE HERE OR PROVIDE LINK TO https://pastebin.com/ OR SIMILAR
```

</details>

## System specifications

<details><summary>System Details</summary>

- OS (with version, e.g Windows 10 v2004 or macOS 10.15 (Catalina)):
- RAM:
- Python version (`python/py/python3 --version`):
- Installed modules (provide output from `pip list`):
```
PASTE HERE
```
</details>

<details><summary>LaTeX details</summary>

+ LaTeX distribution (e.g. TeX Live 2020):
+ Installed LaTeX packages:

</details>

<details><summary>FFMPEG</summary>

Output of `ffmpeg -version`:

```
PASTE HERE
```
</details>

## Additional comments

cc @eulertour

**Comment by e4coder** [comment]
I have opened up a pull request. #1438

**Comment by behackl** [comment]
Closed via #1728.
